lib/prepare.js

```diff
@@ -378,14 +378,18 @@ function updateProjectSplashScreen (platformConfig, locations) {
         'windowSplashScreenAnimatedIcon',
         'windowSplashScreenAnimationDuration',
         'windowSplashScreenBackground',
-        'windowSplashScreenBrandingImage',
+        'android:windowSplashScreenBrandingImage',
         'windowSplashScreenIconBackgroundColor',
         'postSplashScreenTheme'
     ].forEach(themeKey => {
-        const cdvConfigPrefKey = 'Android' + themeKey.charAt(0).toUpperCase() + themeKey.slice(1);
+        let index = 0;
+        if (themeKey.indexOf(':') !== -1) {
+            index = themeKey.indexOf(':') + 1;
+        }
+        const cdvConfigPrefKey = 'Android' + themeKey.charAt(index).toUpperCase() + themeKey.slice(index + 1);
         const cdvConfigPrefValue = platformConfig.getPreference(cdvConfigPrefKey, this.platform);
         let themeTargetNode = splashScreenTheme.find(`item[@name="${themeKey}"]`);
-
+        console.log('hola', cdvConfigPrefKey);
```

Comment:
stray console.log

Reply:
wops, sorry, shouldn't commit so late, I was pretty sure I deleted all the console messages